packages/domain/generators/fe-app/shared/project.json

```diff
@@ -0,0 +1,22 @@
+{
+  "root": "packages/domain/generators/fe-app/shared",
```

Comment:
What is the plan for the generators in here. In general, if we can reduce the number of parameters we pass into the generators the better. Instead of passing in the framework, I would rather two separate generators one for react and one for angular. I think for a lot of the generators we should be able to run them directly without having to go to the nx console and enter the schema options. 
I think the structure would be something like:
frontend
    angular
    react
    shared - this would contain logic that is shared and used in both the react and angular generators

Reply:
Agreed